vmngclient/api/templates/README.md

```diff
@@ -0,0 +1,85 @@
+# How to use
+
+## Feature Template Creation
+
+```python
+omp_vsmart = OMPvSmart(
+    name="my_first_template",
+    description="NA",
+    device_models=[DeviceModel.VSMART]
+    
+)
+
+session.api.templates.create(omp_vsmart)
+```
+
+## Add new Feature Template in vManage-client
+These steps will help you to automate feature template creation. We'll explain what to do and why, so everyone will have deep understanding how our templates work. In the example we will try to create `OMP` Feature Template for vSmart. 
+
+1. Get your template type name and version. Note: display name is not template type name!
```

Comment:
```suggestion
1. Get your template type name and version. 
    >Note: display name is not template type name!
```

Just an aesthetic suggestion, so it would look like that:
![image](https://github.com/CiscoDevNet/vManage-client/assets/85410520/91685520-34b0-4803-b554-3051fb73b5b0)